tfhe/src/integer/server_key/radix_parallel/add.rs

```diff
@@ -801,7 +801,7 @@ impl ServerKey {
         // overhead of dynamic dispatch is negligible compared to multithreading, PBS, etc.
         // we defer all calls to a single implementation to avoid code bloat and long compile
         // times
-        #[allow(clippy::type_complexity)]
+        #[allow(clippy::items_after_statements)]
```

Comment:
what's triggering the lint btw ?

Reply:
Having a function defined in another function with statements before

Reply:
I must be going blind, I don't see the function 😅 below ?